source/Background/utils.js

```diff
@@ -1,15 +1,36 @@
-import { validatePrincipalId, validateAccountId } from '@shared/utils/ids';
+import { validateCanisterId, validatePrincipalId, validateAccountId } from '@shared/utils/ids';
 
 import ERRORS from './errors';
 
+const validateAmount = (amount) => {
+  if (Number.isNaN(amount) || !Number.isInteger(amount) || amount < 0) {
+    return 'Invalid amount. The amount must be a positive integer. \n';
```

Comment:
maybe make this fn return true/false and then when u ask for it return the message 

`const validateAmount = (amount) => Number.isNaN(amount) || !Number.isInteger(amount) || amount < 0`

Reply:
Changed this to keep the function only return `true` of `false` sounds like a better implementation